meta/src/store/storage.rs

```diff
@@ -335,17 +335,34 @@ impl StateMachine {
             let (key, val) = pair?;
             match key.strip_prefix(path.as_str()) {
                 Some(sub_key) => {
-                    if sub_key.find('/').is_some() {
+                    if sub_key.is_empty() {
                         continue;
                     }
-                    if sub_key.is_empty() {
+                    if (path.ends_with("/dbs/")
+                        && !(sub_key.contains("buckets") || sub_key.contains("schemas")))
```

Comment:
I don't think this is a good way. There are too many constant strings here, and it is easy to miss changes during modification.

Reply:
miss "data_nodes" path